src/BinanceApiClient.ts

```diff
@@ -625,6 +673,50 @@ export class BinanceApiClient {
 
     /**
      * Initializes a web socket data stream that gives us information about
+     * Kline/candlestick updates for a number of symbols on the one socket.
+     *
+     * @param symbols           The symbols of which we want to get the candlestick updates.
+     * @param interval          The interval to which the requested candlestick updates
+     *                          refer to.
+     * @param onUpdate          A function to be called when a new update is received.
+     * @param connectionTimeout Timeout based on which the web socket connection is
+     *                          considered to be broken based on a heartbeat monitor.
+     * @param onLostConnection  A callback to be invoked when the web socket connection
+     *                          is detected as broken.
+     */
+     public async monitorCandlesticksCombined(
+         symbols: string[],
+         interval: CandlestickInterval,
+         onUpdate: ( update: CandlestickUpdate ) => any,
+         connectionTimeout?: number,
+         onLostConnection?: () => any ): Promise< void > {
```

Comment:
I'd remove the spaces in the diamond notation here. And it's my bad, I should add a linter to the project.